snap/snapcraft.yaml

```diff
@@ -222,6 +250,13 @@ apps:
     command: bin/support-logging -confdir $SNAP_DATA/config/support-logging/res --registry
     command-chain:
       - bin/security-secret-store-env-var.sh
+    environment:
+      # for support-logging we also want the default persistence to be on a
+      # file so that we can use redis (which isn't supported with support-logging)
+      # in the snap by default
+      Writable_Persistence: 'file'
```

Comment:
Should support logging be removed since all the services have been changed to using just stdout?

Reply:
Well, it's being deprecated, but not actually removed til Hanoi. Trust me, I'd love to remove the old rules engine too, but can't for the same reason.

Reply:
So these services are added to the snap, but disabled?